### ISSUE TITLE
How to play private server with friends?

### ISSUE THREAD

**Issue by efkyxleader** [Barbosik/MultiOgar#371]
pls submit:8

**Comment by MastaCoder** [comment]
...

Port forward..?

**Comment by Jhair2015** [comment]
to me not is posible :( with router

![captura de pantalla 457](https://cloud.githubusercontent.com/assets/18153674/19203099/22c1246a-8c9b-11e6-8ce1-73715bcf064f.png)
![captura de pantalla 458](https://cloud.githubusercontent.com/assets/18153674/19203100/22cbe74c-8c9b-11e6-808c-fed66a737ad4.png)
![captura de pantalla 459](https://cloud.githubusercontent.com/assets/18153674/19203101/22ce660c-8c9b-11e6-90aa-e264c9a44096.png)

**Comment by efkyxleader** [comment]
very very impossible  and how to use skin?

**Comment by efkyxleader** [comment]
(kraken) or <kraken> ?

**Comment by MastaCoder** [comment]
<kraken>

**Comment by NatsuTheGreat** [comment]
Lmao really?

**Comment by MastaCoder** [comment]
@NatsuTheGreat Yeah I had no idea ;D Then it broke my heart that I couldn't use style inside of it :(

**Comment by NatsuTheGreat** [comment]
Yeah lol, it's only what GitHub has here.

<a href="agar.io">click me</a>

**Comment by NatsuTheGreat** [comment]
OH THAT WORKS

**Comment by NatsuTheGreat** [comment]
<h1>Header</h1>

**Comment by NatsuTheGreat** [comment]
<p><h3>Paragraph</h3></p>

**Comment by NatsuTheGreat** [comment]
Lmao awesome, we have more options than what GitHUb gives us here XD

**Comment by cowpits** [comment]
Starting @ 8:00 I explain how you can have your friends join your server. https://www.youtube.com/watch?v=Ng7FsthB1V8

**Comment by cowpits** [comment]
@ Jhair2015 you are very close if you want to port forward. Create a custom service call it OgarServer, plug the port you're using and assign the IP address of the machine that is running the server. All that is explained in my video, how to find which IP you're using, what the rule looks like and how you can find your external IP.  https://www.youtube.com/watch?v=Ng7FsthB1V8 
I run a very busy private server, on average 30 players all the time, 84 players at night. It's busier than real agar.io servers.
